### ISSUE TITLE
The learning rate for ssv2 dataset

### ISSUE THREAD

**Issue by vaesl** [MCG-NJU/VideoMAE#6]
Hi, I have tried to reproduce the VideoMAE performance on SSV2 dataset. I run the experiments on four A100 machines (each includes eight 80G GPUs) and modify --nnodes=4, batch_size 64, such that our total batch size is the same. However, the performance is not consistent with the reported one. I checked the log you provided and notice that the learning rate is different. It seems that your learning rate is not 1.2e-3 (1.5e-4*2048/256) after the warm-up stage. Thanks a lot and I am looking forward to your reply.

**Comment by yztongzhan** [comment]
Hi @vaesl! Thanks for your interest in our VideoMAE. \
1. The base learning rate is set to `1.5e-4` [here](https://github.com/MCG-NJU/VideoMAE/blob/8d7ae2b383261f8f42b62c3ba67f316d87690507/run_mae_pretraining.py#L64-L65). different batch-size can also give favorable results.
2. The lower result is caused by the missing argument `--data_set SSV2` in fine-tuning script. We have fix this issue a few minutes ago :)

**Comment by yztongzhan** [comment]
This issue might lead to a drop of about 1.0%-1.3%. Is it consistent with your results?

**Comment by vaesl** [comment]
> This issue might lead to a drop of about 1.0%-1.3%. Is it consistent with your results?

The performance drop for the 40-th epoch is about 1.0% (64.5%) while the final point drop is about 2.5% (66.7%). Is it similar?

**Comment by yztongzhan** [comment]
We recommend you to fine-tune the pre-trained model again. We have been noticed this issue by e-mail yesterday.  They fine-tune the released model pre-trained for 2400 epochs on SSV2 and got 68.0% (70.3% reported in paper).

**Comment by yztongzhan** [comment]
> This issue might lead to a drop of about 1.0%-1.3%. Is it consistent with your results?

This is 2.0%-2.3%.

**Comment by vaesl** [comment]
OK, I will give a another try. Thanks a lot!

**Comment by liuxingbin** [comment]
> Hi, I have tried to reproduce the VideoMAE performance on SSV2 dataset. I run the experiments on four A100 machines (each includes eight 80G GPUs) and modify --nnodes=4, batch_size 64, such that our total batch size is the same. However, the performance is not consistent with the reported one. I checked the log you provided and notice that the learning rate is different. It seems that your learning rate is not 1.2e-3 (1.5e-4*2048/256) after the warm-up stage. Thanks a lot and I am looking forward to your reply.

hello, I met a problem when i preprocess ssv2 datasets. I run ”ffmpeg -i [input.webm] -c:v libx264 [output.mp4]“ and  get: 
 width not divisible by 2 (427x240).  Have you met this problem? if yes, how you deal with it. Looking forward to your reply.

**Comment by vaesl** [comment]
Hi all,

Based on the codebase updated by @yztongzhan, I have tried to reproduce the reported results and obtained **69.88%** on ssv2 dataset with 800 pretrained epochs. Overall, I feel the learning rate and the total batch size are not that important. One can simply reproduce the reported results based on the updated codebase.

**Comment by yztongzhan** [comment]
Hi @vaesl! Thanks for your update. I'm glad to know that you can successfully reproduce the main results on SSV2.
